.gitignore

```diff
@@ -36,6 +36,7 @@ src/content/*/showcase/_index.yaml
 src/content/*/showcase/*/_toc.yaml
 src/content/*/showcase/*/index.md
 src/content/*/showcase/tags/*
+src/content/*/tools/puppeteer/_src/*
```

Comment:
@petele should I add this? We don't want these files checked into this source but they'll need to end up in g3. Right?

Reply:
Should it go into gulp clean as well?

Reply:
done